packages/grafana-data/src/types/pluginExtensions.ts

```diff
@@ -0,0 +1,11 @@
+// @internal
+export type PluginsExtensionConfig = {
+  links: PluginsExtensionLinkConfig[];
+};
+
+// @internal
+export type PluginsExtensionLinkConfig = {
+  id: string;
+  description: string;
+  path: string;
+};
```

Comment:
Question: similar to the previous one - are we planning to add the `schema` property as well that we outlined in the [design doc](https://docs.google.com/document/d/1-g6-4RJx3LzsldvdwUZb_wHXAS5Z7_kt6Ql9zAoLj8I/edit#)?

Reply:
Yes, but that will be done in later iterations when we have reached a bit further with the schema solution for plugins.